indico/migrations/versions/20200113_1340_6a185fdcd4ee_add_filename_template_column.py

```diff
@@ -0,0 +1,24 @@
+"""Add filename_template column
```

Comment:
Before we merge, I would move this into the existing alembic revision and just paste some SQL in the PR's description for devs to update their DB.

OTOH, we'll probably reset the alembic steps with 3.0 anyway, so it doesn't matter that much.